packages/contracts/hardhat.config.ts

```diff
@@ -69,7 +69,21 @@ task('accounts', 'Prints the list of accounts', async (taskArgs, bre) => {
 })
 
 // Config
-
+let auto: 'auto';
+const default_config = {
```

Comment:
We should use camel case for variable names. I would also suggest a more descriptive name, like `hardhatNetworkDefaultConfig`.

Reply:
Fixed in 172f2cc